docker/snap-breakout/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
```

Comment:
I suggest breaking out the version here and below for easier reusability. 

```bash
VERSION=0.0.1
docker build -t quay.io/humancellatlas/snap-breakout:"${VERSION}" .
docker push quay.io/humancellatlas/snap-breakout:"${VERSION}"
```

Also the below script does not push the container but this one does. Can you pick the one you prefer?